index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <link rel="stylesheet" href="style.css" />
```

Comment:
```suggestion
      <link rel="stylesheet" href="style.css" />
      <meta charset="utf-8">
```
Інакше на деяких системах у вас буде набір символів що не можливо прочитати замість слів